tokio/src/runtime/coop.rs

```diff
@@ -246,14 +246,18 @@ cfg_coop! {
 mod test {
     use super::*;
 
-    #[cfg(tokio_wasm_not_wasi)]
-    use wasm_bindgen_test::wasm_bindgen_test as test;
+    cfg_is_wasm_not_wasi! {
+        use wasm_bindgen_test::wasm_bindgen_test as test;
+    }
+
+    #[cfg(not(all(target_family = "wasm", not(target_os = "wasi"))))]
+    use std::prelude::v1::test;
```

Comment:
```suggestion
    #[cfg(all(target_family = "wasm", not(target_os = "wasi")))]
    use wasm_bindgen_test::wasm_bindgen_test as test;
```

It does not seem reasonable to complicate the code to use cfg_* macro.

(Alternatively, I think you can remove the extra imports (and `self::`) by putting the macro itself after the cfg, rather than adding the cfg to the item,  like [this](https://github.com/taiki-e/portable-atomic/blob/85d04cc3ac369979465aa0967440a43b8a02f4e5/src/utils.rs#L240-L267))

Reply:
Thanks!
I've fixed this according to your suggestion and there are less changes in this PR.

Reply:
> (Alternatively, I think you can remove the extra imports (and `self::`)

I was able to remove the `self::` and reduce the changes to merely change of `cfg`.
It turns out that using `cfg_is_wasm_not_wasi` actually makes rust confused and report "ambiguous use of test".

> by putting the macro itself after the cfg, rather than adding the cfg to the item, like [this](https://github.com/taiki-e/portable-atomic/blob/85d04cc3ac369979465aa0967440a43b8a02f4e5/src/utils.rs#L240-L267))

Not sure this would work since [`wasm_bindgen_test::wasm_bindgen_test`](https://docs.rs/wasm-bindgen-test/latest/wasm_bindgen_test/attr.wasm_bindgen_test.html) is an attribute proc-macro.

Reply:
If my guess is correct, the ambiguity in the import is because the macro handles the input as `:item`. If true, the problem could be avoided by handling the input as `:tt`. And to do that, you need to put the cfg on the macro itself, not on the generated code, as the code I linked does.

Reply:
> If my guess is correct, the ambiguity in the import is because the macro handles the input as `:item`. If true, the problem could be avoided by handling the input as `:tt`. And to do that, you need to put the cfg on the macro itself, not on the generated code, as the code I linked does.

Thanks!

Since I've replaced the use of `cfg_is_wasm_not_wasi!` I don't find much motivation to do this in this PR, although I would be willing to do this in a separate PR.